miner.go

```diff
@@ -139,3 +134,12 @@ func (m *Miner) StopMining() error {
 	}
 	return nil
 }
+
+// Generate makes the CPU miner mine the requested number of blocks
```

Comment:
This would make `StartMining` and `StopMining` obsolete, so we can remove those.